test/plugs/plugs_test.py

```diff
@@ -105,6 +105,7 @@ def test_initialize(self):
         'number': 0
     }},
                      self.plug_manager.as_base_types()['plug_states'])
+    assert AdderPlug.LAST_INSTANCE is not None
```

Comment:
All falsey values are just as dangerous as `NONE`.
```suggestion
    assert AdderPlug.LAST_INSTANCE
```